r01/src/main/resources/templates/fragment/common.html

```diff
@@ -304,20 +304,20 @@
 </textarea>
 
 
-<section id="header" class="clearfix" th:fragment="topFragment">
+    <section id="header" class="clearfix" th:fragment="topFragment">
 
-    <nav class="navbar navbar-default">
+        <nav class="navbar navbar-default">
             <!-- Brand and toggle get grouped for better mobile display -->
             <div class="navbar-header">
                 <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
-                        data-target="#main-navbar-collapse" aria-expanded="false">
+                    data-target="#main-navbar-collapse" aria-expanded="false">
                     <span class="sr-only">Toggle navigation</span>
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
                 <a th:href="@{/}">
-                    <img alt="MindTrails" class="header-logo" th:src="@{/images/logo.png}"/>
```

Comment:
Sure Dan - so sorry about that. I'm not sure what's auto-formatting everything, especially since I've been working in intellij. Do you have an idea what might be causing this, @danfunk ?